### PR TITLE
chore(repo): expand scope of prettier 

### PR DESCRIPTION
**Pull request by rwaskiewicz**
expand the files prettier investigates by looking at all subdirectories
of the listed extensions in our base prettier command. the motivation
for this commit it to prevent needing to "worry" about styling of
`_category_.json` files in PR reviews - I've seen a few of these today
in reviews and called them out as non-blocking. Let's have Prettier tell
us about these.

format the codebase with new rules in a followup commit

### CODE REVIEW

File: static/blog/how-lazy-loading-works.json [hunk 1]
```diff
@@ -1 +1,243 @@
...o building components, while the robots are focused on optimizing everything else, and adjusting to new standards."],"\n",["h2",{"id":"conclusion"},"Conclusion"],"\n",["p",null,"Each framework and bundler has their own way to lazy load modules, and has its own way to be configured, which is great for innovation and new ideas, so we welcome each tool to do it their own way."],"\n",["p",null,"But with Ionic using Stencil’s lazy-loading abilities, which is actually just using native browser APIs, Ionic is able to free itself from each framework’s custom build system, tooling and changes. This helps to meet Ionic’s goal of providing a UI library for any framework, and any version of the framework, or no framework at all, all while generating components that sit atop web-standards. And while core developers can focus on component authoring and maintenance, the compiler can focus on the tasks of optimizing and bundling."],"\n",["p",null,"Hopefully this helps detail is more about how Stencil is addressing the goal of moving the complexity of easily distributing components into the build-time tooling, and let authoring and sharing components easy for developers, and fast for end-users."]]}
```
Comment (rwaskiewicz) [inline static/blog/how-lazy-loading-works.json:1]: I'm going to add something to the backlog to review these old blogs and "do something" with them (extract info and apply to the doc site). ATM, these are inaccessible on our site

File: static/cli.ts [hunk 2]
```diff
@@ -1,2 +1,2 @@
-console.log('\x1b[31mERROR\x1b[0m Stencil for Deno has reached end-of-life')
+console.log('\x1b[31mERROR\x1b[0m Stencil for Deno has reached end-of-life');
```
Comment (rwaskiewicz) [inline static/cli.ts:1]: We still keep this around for Deno support - specifically to say it's EoL

Comment (christian-bromann) [reply static/cli.ts:1]: We have support for Deno? 🤔

**Review by christian-bromann — APPROVED** [review]
👍

File: static/cli.ts (hunk 2 above)
Comment (rwaskiewicz) [reply static/cli.ts:1]: We did, a few years ago. That's part of the reason for the `sys`/`platform` abstraction in Stencil - one for Node, one for Deno, one for the web

**Review by tanner-reits — APPROVED** [review]

**Review by alicewriteswrongs — APPROVED** [review]
all looks good to me 👍

**Pull request merged**